src/main/resources/db/migration/V43_53__Update_ZTJ_and_ZDJ_table.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists zone_tiling_job add column if not exists end_to_end_id varchar;
```

Comment:
End to end ID is used when the ZTJ or ZDJ has to be exposed outside the geo-jobs api. 

Here, it's the full detection ID that must have an end to end it, not the ZTJ or the ZDJ. 

The client doesn't know and SHOULD NOT know, the domain ids, other than the end to end ID.

What you must do then is to just associate the fullDetection entity with the ZTJ and same for ZDJ